src/stabilization_guide.md

```diff
@@ -1,5 +1,10 @@
 # Request for stabilization
 
+**NOTE**: this page is about stabilizing language features.
+For stabilizing library features, see [Stabilizing a library feature].
```

Comment:
Maybe add italics since language and library look somewhat similar?

```suggestion
For stabilizing *library* features, see [Stabilizing a library feature].
```